src/corona_hakab_model/common/medical_state.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, *args, test_willingness: float, **kwargs):
         super().__init__(*args, **kwargs)
         self.test_willingness = test_willingness
 
-        self.contagiousness = BucketDict()
+        self.contagiousness = BucketDict({0:[0]})
```

Comment:
Why you update it here? Let the user type in in consts ...

Reply:
This is a default value, where the state is not contagious (this appears only in immune/susceptible states)